components/Account/Account.jsx

```diff
@@ -8,16 +8,14 @@ import cn from "classnames"
 
 export default function Account({ session }) {
   const supabase = useSupabaseClient()
-  const user = useUser()
```

Comment:
Instead of going through and setting `session.user` everywhere, you could do `const { user } = session`